pkg/beacon/relay/node.go

```diff
@@ -133,7 +135,16 @@ func (n *Node) JoinGroupIfEligible(
 				)
 			}()
 		}
+	} else {
+		go n.RelayMessagesForChannel(channelName)
```

Comment:
Similarly to the entry signing counterpart, I'd name this function `ForwardDKGMessages`